pkg/controller/elasticsearch/driver/driver.go

```diff
@@ -151,6 +151,9 @@ func (d *defaultDriver) Reconcile(ctx context.Context) *reconciler.Results {
 		return results.WithError(err)
 	}
 
+	// Patch the Pods to add the expected node labels as annotations
+	results.WithResults(annotatePodsWithNodeLabels(ctx, d.Client, d.ES))
```

Comment:
Should we mention that we explicitly want to continue reconciliation here even in the presence of errors?